diff --git a/Makefile.am b/Makefile.am
index 4e8f3d1..14aca98 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,4 +1,4 @@
-SUBDIRS = . gdata/tests po docs
+SUBDIRS = . gdata/tests po
 
 ACLOCAL_AMFLAGS = -I m4
 
@@ -447,14 +447,12 @@ gdata_libgdata_la_CPPFLAGS = \
 gdata_libgdata_la_CFLAGS = \
 	$(GDATA_CFLAGS)	\
 	$(GNOME_CFLAGS)	\
-	$(CODE_COVERAGE_CFLAGS)	\
 	$(WARN_CFLAGS)	\
 	$(AM_CFLAGS)
 
 gdata_libgdata_la_LIBADD = \
 	$(GDATA_LIBS)	\
 	$(GNOME_LIBS)	\
-	$(CODE_COVERAGE_LIBS)	\
 	$(AM_LIBADD)
 
 gdata_libgdata_la_LDFLAGS = \
@@ -539,11 +537,6 @@ EXTRA_DIST += gdata/GData-@GDATA_API_VERSION_MAJOR@.@GDATA_API_VERSION_MINOR@.me
 endif
 endif
 
-# Code coverage
-@GNOME_CODE_COVERAGE_RULES@
-CODE_COVERAGE_DIRECTORY = $(top_builddir)/gdata
-CODE_COVERAGE_LCOV_OPTIONS = --base-directory $(abs_top_srcdir)
-
 # Demo programs
 noinst_PROGRAMS =
 
diff --git a/configure.ac b/configure.ac
index 0fb2fee..0c1edcf 100644
--- a/configure.ac
+++ b/configure.ac
@@ -133,9 +133,6 @@ AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, ["$GETTEXT_PACKAGE"], [Define to the Gettext
 AC_SUBST(GETTEXT_PACKAGE)
 IT_PROG_INTLTOOL([0.40.0])
 
-# Code coverage
-GNOME_CODE_COVERAGE
-
 # General GNOME macros
 GNOME_DEBUG_CHECK
 GNOME_COMPILE_WARNINGS([maximum])
@@ -150,16 +147,11 @@ AC_SUBST([AM_CFLAGS])
 AC_SUBST([AM_CXXFLAGS])
 AC_SUBST([AM_LDFLAGS])
 
-GTK_DOC_CHECK([1.14],[--flavour no-tmpl])
-
 AC_CONFIG_FILES([
 Makefile
 libgdata.pc
 gdata/gdata-version.h
 gdata/tests/Makefile
 po/Makefile.in
-docs/Makefile
-docs/reference/Makefile
-docs/reference/version.xml
 ])
 AC_OUTPUT
